IHP/IDE/CodeGen/ApplicationGenerator.hs

```diff
@@ -133,22 +131,28 @@ generateGenericApplication applicationName =
                 <> "</body>\n"
                 <> "|]\n"
                 <> "\n"
+                <> "cssfiles = do\n"
+                <> "    when (isDevelopment FrameworkConfig.environment) [hsx|\n"
+                <> "      <link rel=\"stylesheet\" href=\"/vendor/bootstrap.min.css\"/>\n"
+                <> "      <link rel=\"stylesheet\" href=\"/vendor/flatpickr.min.css\"/>\n"
+                <> "    |]\n"
+                <> "    [hsx| <link rel=\"stylesheet\" href=\"/app.css\"/> |]\n"
```

Comment:
Usually the `app.css` is not the bundled css. [There is the `prod.css` file which can be built using `make`](https://github.com/digitallyinduced/ihp/blob/master/lib/IHP/Makefile.dist#L134). I guess we need to include the `prod.css` here in production mode, and the `app.css` only in dev.